src/ipc-handlers.ts

```diff
@@ -560,6 +561,11 @@ export async function saveOnboarding(
 	} );
 }
 
+export async function executeWPCLiInline( _event: IpcMainInvokeEvent, args: string[] ) {
```

Comment:
I wonder if the args should be just a `string` and we can split the spaces to convert them into an array in this function 🤔 .
Curious about what others think.